concent-builder/deploy.yml

```diff
@@ -0,0 +1,36 @@
+- hosts:
+    - concent-builder
+  vars_files:
+    - consts.yml
+  tasks:
+    - become:      yes
+      remote_user: builder
+      block:
+        - name:    Google Cloud Engine service account activating
+          command: gcloud auth activate-service-account concent-builder-service@devel-concent-golem-network.iam.gserviceaccount.com --key-file "/home/{{ shared_user }}/build/concent-secrets/devel-concent-golem-network-198b161e7c8a.json"
```

Comment:
1) You can split long lines in YAML like this:

      ``` yaml
      - name:    Activate Google Container Engine service account
        command: gcloud auth activate-service-account                                   \
            concent-builder-service@devel-concent-golem-network.iam.gserviceaccount.com \
            --key-file "/home/{{ shared_user }}/build/concent-secrets/devel-concent-golem-network-198b161e7c8a.json"
      ```

2) You should use variables you define. `/home/{{ shared_user }}/build` is `{{ build_dir }}`. Or better, define `{{ secret_dir }}`.

3) Let's put the name of the account in a variable in `vars.yml`. Values like this should never be hard-coded in this repository. That's what `concent-deployment-values` is for.